documentation/sphinx/source/tls.rst

```diff
@@ -29,10 +29,50 @@ This will configure the new cluster to communicate with TLS.
 
 .. note:: Depending on your operating system, version and configuration, there may be a firewall in place that prevents external access to certain ports. If necessary, please consult the appropriate documentation for your OS and ensure that all machines in your cluster can reach the ports configured in your :ref:`configuration file <foundationdb-conf>`.
 
-.. _converting-existing-cluster:
+.. _converting-existing-cluster-after-6.1:
 
-Converting an existing cluster to use TLS
-=========================================
+Converting an existing cluster to use TLS (since v6.1)
+======================================================
+
+Since version 6.1, FoundationDB clusters can be converted to TLS without downtime. FoundationDB server can listen to TLS and unencrypted traffic simultaneously on two separate ports. As a result, FDB clusters can live migrate to TLS:
+
+1) Restart each FoundationDB server individually, but with an additional listen address for TLS traffic::
+
+     /path/to/fdbserver -C fdb.cluster -p 127.0.0.1:4500 -p 127.0.0.1:4600:tls
+
+   Since, the server still listens to unencrypted traffic and the cluster file still contains the old address, rest of the processes will be able to talk to this new process.
+
+2) Once all processes are listening to both TLS and unencrypted traffic, switch one or more coordinator to use TLS. Therefore, if the old coordinator list was ``127.0.0.1:4500,127.0.0.1:4501,127.0.0.1:4502``, the new one would be something like ``127.0.0.1:4600:tls,127.0.0.1:4501,127.0.0.1:4502``. Switching few coordinators to TLS at a time allows a smoother migration and a window to find out clients who do not yet have TLS configured. The clients which cannot connect to TLS coordinators are listed in ``fdbstatus`` (look for ``tls_configured`` field in ``clients``)::
+
```

Comment:
Maybe add another paragraph before 3): 
If there exist a client that cannot connect to all coordinators after a coordinator is switched to TLS, it mean the client does not set up its TLS correctly. System operator should notify the client to correct the client's TLS configuration. 
Otherwise, when all coordinators are switched to TLS ports, the client will loose connection.

Reply:
I will just suggest the change then.

Reply:
Thanks! I added that part to the new changes.